pkg/test/utils/kubernetes.go

```diff
@@ -1057,8 +1067,6 @@ func RunCommands(logger Logger, namespace string, commands []harness.Command, wo
 	}
 
 	for _, cmd := range commands {
-		logger.Logf("running command: %q", cmd.Command)
```

Comment:
why is this removed... this keeps coming and going... and was resolved with marcin a couple of weeks ago.  It likely we need to review those closed PRs.